README.md

```diff
@@ -27,7 +29,6 @@ Version  | Codename  | LTS
 3.5      | Kirkstone | Yes
 3.4      | Honister  | No
 3.3      | Hardknott | No
-3.1      | Dunfell   | Yes
```

Comment:
There is still a working dunfell branch. Maybe reword "The current layer is compatible with the following Yocto version:" by stating that this table refers to master branch

Reply:
Good hint.